src/Basics/Extra.elm

```diff
@@ -54,7 +54,7 @@ smaller than that, and behaviour becomes mathematically unsound.
 -}
 minSafeInteger : number
 minSafeInteger =
-    -9007199254740991
+    - maxSafeInteger
 
```

Comment:
Oh my, I'm so sorry. Glad you noticed! Disallowing a space for unary negation seems alright to me, and some googling reveals [it was very much intentional](https://groups.google.com/forum/#!topic/elm-discuss/DcvoUKPzM_M). Thanks for the fix, @jvoigtlaender. Reminds me to check if there was ever an issue logged for the integer truncation, tho.